controllers/util.go

```diff
@@ -78,7 +78,7 @@ func addFinalizers(ctx context.Context, client client.Client, resource client.Ob
 	if err := client.Update(ctx, resource); err != nil {
 		return defaultRequeue, err
 	}
-	return fastRequeue, nil
+	return ctrl.Result{}, nil
```

Comment:
🤔 is this because issuing the update the line before triggers the controller to reconcile anyway?

Reply:
yes